src/Matterhorn/FilePaths.hs

```diff
@@ -87,7 +94,9 @@ bundledEmojiJsonPath = do
                  -- In this case we assume the binary is being run from
                  -- a release, in which case the syntax directory is a
                  -- sibling of the executable path.
-                 takeDirectory selfPath </> emojiDirName </> emojiJsonFilename
+                 fromMaybe
+                     (takeDirectory selfPath </> emojiDirName </> emojiJsonFilename)
+                     dataEmojiJsonPath
```

Comment:
Please flip the order on this `fromMaybe` so that the preferred path is the previously-used one and the fallback path is `dataEmojiJsonPath`.

Reply:
Fair enough, done.